build/build-release.yml

```diff
@@ -88,14 +88,14 @@ steps:
       command: 'custom'
       custom: 'pack'
       arguments: '--no-build -c Release -o packages -p:ContinuousIntegrationBuild=true'
-      projects: src/*/WebJobs.Extensions.OpenAI*.csproj
+      projects: src/*/WebJobs.Extensions.OpenAI*.csproj | src/*/Functions.Worker.Extensions.OpenAI*.csproj
```

Comment:
src/*/*Extensions.OpenAI*.csproj

Reply:
Discussed offline and we will do this change in a seperate PR